apps/taxes/src/modules/avatax/ui/avatax-configuration-credentials-fragment.tsx

```diff
@@ -55,107 +55,51 @@ export const AvataxConfigurationCredentialsFragment = (
   return (
     <>
       <FormSection title="Credentials">
-        <Box paddingY={4} display={"flex"} flexDirection={"column"} gap={10}>
-          <div>
-            <Input
-              control={control}
-              name="credentials.username"
-              required
-              label="Username *"
-              helperText={formState.errors.credentials?.username?.message}
-            />
-            <HelperText>
-              You can obtain it in the <i>API Keys</i> section of <i>Settings</i> → <i>License</i>{" "}
-              in your Avalara Dashboard.
-            </HelperText>
-          </div>
-          <div>
-            <Input
-              control={control}
-              name="credentials.password"
-              type="password"
-              required
-              label="Password *"
-              helperText={formState.errors.credentials?.password?.message}
-            />
-            <HelperText>
-              You can obtain it in the <i>API Keys</i> section of <i>Settings</i> → <i>License</i>{" "}
-              in your Avalara Dashboard.
-            </HelperText>
-          </div>
+        <div>
+          <Input
+            control={control}
+            name="credentials.username"
+            required
+            label="Username *"
+            helperText={formState.errors.credentials?.username?.message}
+          />
+          <HelperText>
+            You can obtain it in the <i>API Keys</i> section of <i>Settings</i> → <i>License</i> in
+            your Avalara Dashboard.
+          </HelperText>
+        </div>
+        <div>
+          <Input
+            control={control}
+            name="credentials.password"
+            type="password"
+            required
+            label="Password *"
+            helperText={formState.errors.credentials?.password?.message}
+          />
+          <HelperText>
+            You can obtain it in the <i>API Keys</i> section of <i>Settings</i> → <i>License</i> in
+            your Avalara Dashboard.
+          </HelperText>
+        </div>
 
-          <div>
-            <Input
-              control={control}
-              name="companyCode"
-              label="Company name"
-              helperText={formState.errors.companyCode?.message}
-            />
+        <AppToggle
+          control={control}
+          label="Use sandbox mode"
+          helperText={
             <HelperText>
-              When not provided, the default company will be used.{" "}
+              Toggling between{" "}
```

Comment:
**Suggestion[^1][^2]**:
The helper might be a bit confusing: according to docs prod and stage have a separate credentaials, so this input is not "Toggling", but should be set accordingly to the type of credentials.

[^1]: ✅ Not blocking the review. 
[^2]: It might have an impact on the submitted code. I've probably had a similar issue with the chosen approach. If you're sure that's overkill or does not apply to this PR, let me know and resolve the request.